Sources/Hummingbird/Application.swift

```diff
@@ -102,7 +102,7 @@ extension ApplicationProtocol {
             eventLoopGroup: self.eventLoopGroup,
             logger: self.logger
         ) { (request, responseWriter: consuming ResponseWriter, channel) in
-            let logger = self.logger.with(metadataKey: "hb_id", value: .stringConvertible(RequestID()))
+            let logger = self.logger.with(metadataKey: "hb.id", value: .stringConvertible(RequestID()))
```

Comment:
```suggestion
            let logger = self.logger.with(metadataKey: "http.request.id", value: .stringConvertible(RequestID()))
```

Reply:
Non standard and we don't own `http` namespace. Using `hb.request.id`